.github/workflows/auto_translate.yml

```diff
@@ -0,0 +1,43 @@
+name: Auto translate
+on:
+  workflow_call:
+    inputs:
+      branch:
+        type: string
+        default: ${{ github.ref }}
+      locale_folder:
+        type: string
+        default: "locale"
+
+jobs:
+  autotranslate:
+    env:
+      API_KEY: ${{secrets.DL_API_KEY}}
+      LOCALE_FOLDER: ${{ github.workspace }}/action/package/${{ inputs.locale_folder }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.branch }}
+          path: action/package/
+      - name: Checkout Scripts Repo
+        uses: actions/checkout@v3
+        with:
+          repository: OpenVoiceOS/.github
+          path: action/github/
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - name: Install Translate Tools
+        run: |
+          python -m pip install ovos-translate-plugin-deepl ovos-utils
```

Comment:
Recommend pulling from a requirements.txt in the scripts folder instead

Also recommend having an additional line that installs arbitrary Python packages, as again I expect some repo will need to do that